Sources/Reporters/SmartInsightsReporter.swift

```diff
@@ -43,12 +43,19 @@ public struct SmartInsightsReporter: DiagnosticsReporting {
     init() {
         var defaultInsights: [SmartInsightProviding?] = [
             DeviceStorageInsight(),
-            UpdateAvailableInsight()
         ]
         #if os(iOS) && !targetEnvironment(macCatalyst)
             defaultInsights.append(CellularAllowedInsight())
         #endif
         
+        #if os(iOS)
+        if #available(iOS 13, *) {
+            defaultInsights.append(UpdateAvailableInsight())
+        }
+        #else
+        defaultInsights.append(UpdateAvailableInsight())
```

Comment:
A comment explaining this logic might be helpful, as reading this was a bit confusing — I'm assuming `UpdateAvailableInsight` is not available on iOS < 13, but it is available on all other platforms in their respective supported versions?

Reply:
Yep, makes sense. But I think we can follow your suggestion to make UpdateAvailableInsight() iOS11 compatible instead. Then we can revert this logic.